app/services/expired/dossiers_deletion_service.rb

```diff
@@ -26,17 +27,20 @@ def send_brouillon_expiration_notices
       .without_brouillon_expiration_notice_sent
       .limit(BROUILLON_DELETION_EMAILS_LIMIT_PER_DAY)
 
-    user_notifications = group_by_user_email(dossiers_close_to_expiration)
+    dossiers_close_to_expiration.includes(:user, :procedure).find_in_batches(batch_size: BATCH_SIZE) do |batch|
+      user_notifications = batch
```

Comment:
idée d'amélioration car ça me chagrine cet ordre du batch. 

Je m'explique : nous batchions pour mettre dans un mail tous les dossiers qui seront supprimé pour un usager. Le fait de wrapper ça ds un batch de 1000, couplé à l'ordonnancement dossier.id, on vient un peu chambouler cette logique. 

Pour des raisons de perf, je suis aligné, question : et si on ordonnait par user_id ? On pourra envoyer 2x le mail si l'overlap est dommage, mais cette aproche limite ce risque non ? 

Sinon faudrait envoyer les mails en itération n+1 mais ca commence a faire lourd j'imagine